pdf/pdfrenderthread.cpp

```diff
@@ -214,8 +220,14 @@ void PDFRenderThreadQueue::processPendingJob()
         return;
 
     PDFJob* job = dequeue();
-    if (job->type() != PDFJob::LoadDocumentJob)
-        job->m_document = d->document;
+    switch(job->type())
+    {
+        case PDFJob::LoadDocumentJob: {
+            d->loadFailure = false;
+        }
+        default:
+            job->m_document = d->document;
+    }
```

Comment:
This project is probably not following the Qt conventions in other places, but we aim follow them, see here: http://qt-project.org/wiki/Qt_Coding_Style#e289ee44592e9c32d4212069f0806daf